src/Adapter/Shop/QueryHandler/SearchShopsHandler.php

```diff
@@ -43,14 +45,21 @@ final class SearchShopsHandler implements SearchShopsHandlerInterface
      */
     private $shopRepository;
 
+    /**
+     * @var ShopGroupRepository
+     */
+    private $shopGroupRepository;
+
     /**
      * SearchShopsHandler constructor.
      *
      * @param ShopRepository $shopRepository
+     * @param ShopGroupRepository $shopGroupRepository
      */
-    public function __construct(ShopRepository $shopRepository)
+    public function __construct(ShopRepository $shopRepository, ShopGroupRepository $shopGroupRepository)
```

Comment:
Isn't a BC ?

Reply:
it's a new class from develop, si it has never been released yet ;)

Reply:
Thanks for that.

I close this review